example/ReflectionExample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
Since BugTrackerExample already exports to DOT graph, could we just tweak that one rather than add a new one to demonstrate reflection capabilities? https://github.com/dotnet-state-machine/stateless/blob/dev/example/BugTrackerExample/Program.cs#L23